api/src/glfs-fops.c

```diff
@@ -1513,6 +1516,7 @@ glfs_pwritev_common(struct glfs_fd *glfd, const struct iovec *iovec, int iovcnt,
                     struct glfs_stat *poststat)
 {
     xlator_t *subvol = NULL;
+    xlator_t *this = THIS;
```

Comment:
`THIS` at this point will likely be invalid. It cannot be used until it has been correctly initialized after line 1537.

This also happens in many other functions.

Reply:
Done